src/update.cr

```diff
@@ -6,7 +6,8 @@ module Update
     Update::Providers::APM,
     Update::Providers::NPM,
     Update::Providers::Gems,
-    Update::Providers::AppStore
+    Update::Providers::AppStore,
+    Update::Providers::Composer
```

Comment:
I am going to move this line above the App Store so that Composer packages are updated with the other language packages though